libs/features/georoutingv2/georoutingv2.js

```diff
@@ -166,6 +166,15 @@ function openPicker(button, locales, country, event, dir) {
     list.appendChild(li);
   });
   button.parentNode.insertBefore(list, button.nextSibling);
+  const buttonRect = button.getBoundingClientRect();
+  const windowHeight = window.innerHeight;
+  const spaceBelowButton = windowHeight - buttonRect.bottom;
+
+  if (spaceBelowButton <= list.offsetHeight) {
+    list.style.bottom = `${buttonRect.height}px`;
```

Comment:
the height of the button is constant no matter the size of the viewport (something like 41px). You can therefore set the `bottom` css also over the `.top` class.